.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     build:
         working_directory: ~/repo
         docker:
-            - image: circleci/python:3.7
```

Comment:
this is another set of changes that should be on their own commit
this would be a good opportunity for you to learn to rewrite your changes as separate commits, so that the commit that adds google cloud as an asr provider doesn't have any of this stuff in it